yarl/__init__.py

```diff
@@ -204,6 +204,8 @@ def build(
             raise ValueError('Can\'t build URL with "port" but without "host".')
         if query and query_string:
             raise ValueError('Only one of "query" or "query_string" should be passed')
+        if path is None or query_string is None or fragment is None:
```

Comment:
How about
```suggestion
        if None in (path, query_string, fragment):
```

Reply:
I think better to have a separate check for every argument.
It can help with constructing more informative error text.
`'NoneType not accepted. Use "".'` looks too cryptic, a parameter name for the failed check can improve a user experience a lot.

Reply:
true